spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb

```diff
@@ -39,6 +39,11 @@ module AwsRefresherSpecCommon
   ].freeze
 
   def stub_refresh_settings(settings)
+    # TODO(lsmola) extract the batch sizes to the settings and stub the settings instead
+    # Lower batch sizes to test multiple batches for each collection
+    allow_any_instance_of(ManagerRefresh::InventoryCollection).to receive(:batch_size).and_return(4)
+    allow_any_instance_of(ManagerRefresh::InventoryCollection).to receive(:batch_size_pure_sql).and_return(4)
```

Comment:
I dislike `allow_any_instance_of` this is a good reason why the batch size should be configurable